cmd/stack/deploy.go

```diff
@@ -66,3 +70,19 @@ func Deploy(ctx context.Context) *cobra.Command {
 	cmd.Flags().BoolVarP(&noCache, "no-cache", "", false, "do not use cache when building the image")
 	return cmd
 }
+
+func printDisplayPorts(s *model.Stack) {
+
+	for name, svc := range s.Services {
+		if len(svc.Ports) > 0 {
+			log.Println(fmt.Sprintf("    %s ", log.BlueString(name)))
+			for _, p := range svc.Ports {
```

Comment:
Ports are not public or private, what is public is the service.
If we are in an Okteto Cluster, we should show the full endpoints here.
I would leave this part for later, and create an issue to track it